framework/kms-keyring.md

```diff
@@ -5,10 +5,15 @@
 
 ## Version
 
-0.2.1
+0.2.2
 
 ### Changelog
 
+- 0.2.2
+
+  - Rename Key IDs to [Key Names](#key-names) for increased clarity
```

Comment:
I know why you did not link to the issues,
but should we at least link to the PR?
I know we can get it from source, but it is nice
and easy to have it handy right in the doc?